.github/CODEOWNERS

```diff
@@ -1,2 +1,5 @@
 # Default owners
-*       @hashicorp/hds-engineering
\ No newline at end of file
+*       @hashicorp/hds-engineering
+
+# co-ownership of website docs
+/website/docs/ @hashicorp/hds-engineering @hashicorp/hds-design
```

Comment:
should we include also the `website/public/assets` folder? usually they add images for the documentation there, too

Reply:
good call. I added that as well.